astrosat_users/models/models_customers.py

```diff
@@ -40,6 +40,24 @@ def multiple(self):
         return self.filter(customer_type=CustomerType.MULTIPLE)
 
 
+class CompanyTypes(models.TextChoices):
+    NON_PROFIT = 'NON_PROFIT', _('Non-Profit Organisation')
+    LOCAL_AUTHORITY = 'LOCAL_AUTHORITY', _('Local Authority')
+    GOV_AND_EXEC_AGENCIES = 'GOV_AND_EXEC_AGENCIES', _(
+        'Government & Executive Agencies'
+    )
+    NON_DEPT_PUBLIC_BODY = 'NON_DEPT_PUBLIC_BODY', _(
+        'Non Departmental Public Body'
+    )
+    PUBLIC_CORP = 'PUBLIC_CORP', _('Public Corporation')
+    HEALTH_AND_CARE = 'HEALTH_AND_CARE', _('Health & Care')
+    CHARITY = 'CHARITY', _('Charity')
+    ACADEMICS_OR_EDUCATION = 'ACADEMICS_OR_EDUCATION', _(
+        'Academics, School or any kind of Education'
+    )
+    OTHER = 'OTHER', _('Other')
+
+
```

Comment:
I would move the definition of `CompanyTypes` to be w/ the other enumerations.  Or, if you prefer, define it inside the `Customer` class itself.

Reply:
Which would you prefer? I put it there as it is generally good to have related code together, but I'm not against placing it with other enums. I guess, I would really prefer to see it inside the `Customer` class itself, as I don't see it can exist outwith a customer. I assume that would mean just adding this class after the `Meta` class of `Customer`.